layouts/blog/list.html

```diff
@@ -15,9 +15,46 @@ <h3>Categories</h3>
             {{ range $pages }}
                 {{ .Render "summary" }}
             {{ end }}
-            {{ template "_internal/pagination.html" . }}
+            
+            {{ $page := .Paginator }}
+            {{ if gt $page.TotalPages 1 }}
+            {{ $.Scratch.Set "dot_rendered" false }}
+            <nav aria-label="page navigation">
+                <ul class="pagination">
+                    <!-- Don't show on 1st and 2nd page -->
+                    {{ if and (ne $page.PageNumber 1) }}
```

Comment:
Perhaps

```suggestion
                    {{ if not .$page.First }}
```

Reply:
The command you suggested isn't performing. **not $page.First** doesn't provide any output.